db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
```

Comment:
we don't want these file changes in `master`.
let me know if you want to reset this file to match `master` or you want me to do it! :)